sugar-iconify.md

```diff
@@ -0,0 +1,143 @@
+#Sugar Iconify
+
+##Description
+
+This script will create Sugar-compatible SVG icons from input.svg by
```

Comment:
I think you need to start by saying something about how Sugar uses entities for its SVG colors and that there is a script, sugariconfiy.py that will insert entities into an SVG file. Then explain that the rest of this document is about the script: where to find it and how to use it. You can also point the reader to the Sugar guidelines that describe the Sugar color scheme in more detail